packages/node/src/behaviors/power-source/PowerSourceServer.ts

```diff
@@ -5,13 +5,38 @@
  */
 
 import { DescriptorServer } from "#behaviors/descriptor";
+import { ClusterType } from "@matter/types";
+import { PowerSource } from "@matter/types/clusters/power-source";
 import { PowerSourceBehavior } from "./PowerSourceBehavior.js";
 
+const PowerSourceLevelBase = PowerSourceBehavior.with(PowerSource.Feature.Battery, PowerSource.Feature.Rechargeable);
+
 /**
  * This is the default server implementation of {@link PowerSourceBehavior}.
  */
-export class PowerSourceServer extends PowerSourceBehavior {
+export class PowerSourceServerLogic extends PowerSourceLevelBase {
     override async initialize() {
         (await this.agent.load(DescriptorServer)).addDeviceTypes("PowerSource");
+
+        // According to specs changes to these attributes should not occur more often than every 10 seconds
+        if (this.events.batPercentRemaining$Changed !== undefined) {
+            this.events.batPercentRemaining$Changed.quiet.config = {
+                minimumEmitIntervalMs: 10_000,
+            };
+        }
+        if (this.events.batTimeRemaining$Changed !== undefined) {
+            this.events.batTimeRemaining$Changed.quiet.config = {
```

Comment:
QuietObservable implements the config interface so you can do foo.quiet.minimumCommitIntervalMS = 10_000